tools/build.js

```diff
@@ -77,7 +77,14 @@ commander
 
 commander.target = commander.target.toLowerCase();
 
-build     = require(`./${commander.target}`);
+var target = commander.target
+if (target=="cdn")
+  target = "./rollup_cdn"
+if (target=="browser")
+  target = "./rollup_browser"
+
+
+build     = require(`./${target}`);
```

Comment:
I guess because I was still referencing them.  I don't mind renaming them though for the final version of this.

Reply:
> And what about language definitions in separate repos, are they also supposed to use it?

I still need to review that.  It might make more sense for languages to use commons require format (making them usable with both node.js directly AND the build system - maximum flexibility).  I was going to play around and see what worked best.

But yes the idea is to come up with a common standard and then the blessed way to build a monolithic build would be if all the packages followed the preferred standard. 

This is still WIP. :-)  

> Is roll.js used at all? It looks like a first attempt that wasn't deleted.

Yeah, more WIP.  This hasn't been cleaned up yet.